scripts/asm-installer/install_asm

```diff
@@ -1411,6 +1440,23 @@ is_workload_identity_enabled() {
   if [[ "${ENABLED}" = 'null' ]]; then false; fi
 }
 
+get_hub_workload_identity() {
+  local ID_NUMBER
+  ID_NUMBER=`kubectl get membership -o=json | jq '.items | length'`
+  if [[ ${ID_NUMBER} -gt 1 ]]; then
+    fatal "Found more than one Hub Membership. Please make sure one cluster is associated with one Hub Membership.";
+  fi
+  local IDENTITY_PROVIDER
+  IDENTITY_PROVIDER=`kubectl get membership -o=json | jq .items[0].spec.identity_provider`
+  if [[ ${IDENTITY_PROVIDER} =~ ^\"https:\/\/gkehub.googleapis.com\/projects\/(.+)\/locations\/global\/memberships\/(.+)\"$ ]]; then
+    ENVIRON_PROJECT_ID=${BASH_REMATCH[1]}
+    HUB_MEMBERSHIP_ID=${BASH_REMATCH[2]}
```

Comment:
It looks like CI isn't really working properly right now, but I don't think this is posh compatible. We try to avoid bash specific functionality in case someone wants to use busybox or something in automation.

You should be able to do something like

```
read -r ENVIRON_PROJECT_ID HUB_MEMBERSHIP_ID <(echo "${IDENTITY_PROVIDER}" | sed 's/^\"https:\/\/gkehub.googleapis.com\/projects\/(.+)\/locations\/global\/memberships\/(.+)\"$/\1 \2/g')
```